breaking-front/src/hooks/queries/useLogout.js

```diff
@@ -1,19 +1,18 @@
+import { useNavigate } from 'react-router-dom';
+import { useQuery, useQueryClient } from 'react-query';
 import { getLogout } from 'api/login';
 import { PAGE_PATH } from 'constants/path';
-import { useQuery, useQueryClient } from 'react-query';
-import { useNavigate } from 'react-router-dom';
 
 const useLogout = () => {
   const queryClient = useQueryClient();
   const navigate = useNavigate();
-
   return useQuery(['logout'], getLogout, {
     enabled: false,
     onSuccess: () => {
       alert('로그아웃 되었습니다.');
       localStorage.removeItem('access_token');
       //refresh 토큰 또한 제거 예정
-      queryClient.clear();
+      queryClient.invalidateQueries('jwtValidate');
```

Comment:
```suggestion
      const useLogout = () => {
  const queryClient = useQueryClient();
  const navigate = useNavigate();
  const userInformation = useContext(UserInformationContext);
  return useQuery(['logout'], getLogout, {
    enabled: false,
    onSuccess: () => {
      alert('로그아웃 되었습니다.');
      localStorage.removeItem('access_token');
      //refresh 토큰 또한 제거 예정
      queryClient.clear();
      userInformation.setUserInformation({});
      navigate(PAGE_PATH.LOGIN);
    },
  });
};
```

로그아웃시에 캐시또한 전부 날려야 되므로 이게 더 의도에 부합한것 같습니다.